distrobox-create

```diff
@@ -41,6 +42,7 @@ Options:
 
 	--image/-i:		image to use for the container	default: registry.fedoraproject.org/fedora-toolbox:35
 	--name/-n:		name for the distrobox		default: fedora-toolbox-35
+	--duplicate/-d:		name of the distrobox container to use as base for a new container
```

Comment:
I'd prefer the name `clone`. What do you think?

Reply:
mmh yea probably it's a bit more pertinent as a name